library/libraries.js

```diff
@@ -437,6 +437,20 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
         }
     },
 
+    '$mol_view': {
```

Comment:
```suggestion
    '$mol': {
```
Since the documentation for this library refers to itself as `$mol`, the name should reflect that.

Reply:
`$mol` is the namespace. There are many libraries. `$mol_view` (view components base class) is one of them.